hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java

```diff
@@ -124,12 +134,55 @@ private Event getApiEntrySpanForExitSpan(
         .orElse(null);
   }
 
+  /**
+   * An api_node represents an api call in the trace
+   * It consists of api_entry span and multiple api_exit and internal spans
+   * <p>
+   * This method computes the count of exit calls for any given api (identified by api_entry_span)
+   * This count is a composition of 2 things
+   * 1. link between api_exit_span in api_node to api_entry_span (child of api_exit_span) in another api_node
+   * 2. exit calls to backend from api_exit_span in api_node
+   */
+  Map<ByteBuffer, Integer> computeApiExitCallCount(StructuredTrace trace) {
```

Comment:
General note on refactoring later:
ApiTraceGraph is defined hypertrace-trace-enricher api, [ApiNodeEventEdge](https://github.com/hypertrace/data-model/blob/main/data-model/src/main/avro/ApiNodeEventEdge.avdl) is defined as avro data model, and uses some indexing on ` int src_api_node_index;` on ApiTraceGraph, it was really hard to understand. We need to rethink this set of data-structures. I went back, and forth to understand this. Logically, looks fine to me.